packages/runtime/src/useCases/transport/account/LoadItemFromTruncatedReference.ts

```diff
@@ -93,7 +93,7 @@ export class LoadItemFromTruncatedReferenceUseCase extends UseCase<LoadItemFromT
         const tokenContent = token.cache.content;
 
         if (tokenContent instanceof TokenContentRelationshipTemplate) {
-            const template = await this.templateController.loadPeerRelationshipTemplate(tokenContent.templateId, tokenContent.secretKey, tokenContent.forIdentity?.toString());
+            const template = await this.templateController.loadPeerRelationshipTemplateByTokenContent(tokenContent);
```

Comment:
Making this use case clean requires that tokens for password-protected templates are protected with the same password - I'd like to only implement this in the password-protected tokens PR.

Reply:
This means we have to merge password-protected tokens first?

Reply:
Maybe it's enough to release both things together as this PR seems to come to an end.